package.json

```diff
@@ -655,6 +655,11 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Enables the Advanced Web App Creation which will prompt for app name, location, website OS, runtime, resource group, and app service plan."
+                },
+                "appService.enableConnectionsNode": {
```

Comment:
Please remove this section until the "Connections" feature actually provides value to the user. This is the correct thing to do if the feature was in preview, but this actually isn't even ready for "preview" support.

You should see this in your settings.json:
![screen shot 2018-09-06 at 9 40 59 am](https://user-images.githubusercontent.com/11282622/45172069-0b36b180-b1b9-11e8-9340-9eb601f7efea.png)

But it should still work - which is all we need for our own testing purposes.